index.html

```diff
@@ -2,9 +2,130 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Business Website Assignment</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./styles/styles.css">
 </head>
 <body>
-
+    <header>
+        <div class="outer-container">
+            <nav class="inner-container-large-nav">
+                <ul>
+                    <li>Case Studies</li>
+                    <li>Services</li>
+                    <li>Contact Us</li>
+                </ul>
+            </nav>
+        </div>
+        <div class="outer-container">
+            <div class="inner-container-small introblock">
+                <h1>Your website is an investment in the  success of your business</h1>
+                <p>We are here to help you succeed</p>
+                <button>LET'S WORK TOGETHER</button>
+                <img src="assets/squiggle_line_stripe.svg" alt="------" />
+            </div>
+        </div>
+    </header>
+    <main>
+        <section class="outer-container">
+            <div class="inner-container-small">
+                <div class="intro-inner-container">
```

Comment:
2 inner containers is overbodig. Mocht je extra styling nodig hebben voor de inner container, dan is het beter om een extra class toe te voegen.

`<div class="intro-inner-container extra-class">`